plaidml/core/__init__.py

```diff
@@ -164,7 +164,8 @@ def sizes(self):
     @property
     def strides(self):
         return [
-            ffi_call(lib.plaidml_shape_get_dim_stride, self.as_ptr(), i) for i in range(self.ndims)
+            ffi_call(lib.plaidml_shape_get_dim_stride, self.as_ptr(), i)
+            for i in range(self.ndims)
```

Comment:
I'm wondering how these changes got in, `yapf` should be making this all uniform and I don't see any other change other than style here.

Reply:
resolved using correct yapf version